quarkus/service/src/main/java/org/apache/polaris/service/quarkus/metrics/QuarkusValueExpressionResolver.java

```diff
@@ -22,16 +22,16 @@
 import io.micrometer.common.lang.Nullable;
 import jakarta.annotation.Nonnull;
 import jakarta.enterprise.context.ApplicationScoped;
-import org.apache.polaris.core.context.RealmContext;
+import org.apache.polaris.core.context.RealmId;
 
 @ApplicationScoped
 public class QuarkusValueExpressionResolver implements ValueExpressionResolver {
 
   @Override
   public String resolve(@Nonnull String expression, @Nullable Object parameter) {
     // TODO maybe replace with CEL of some expression engine and make this more generic
-    if (parameter instanceof RealmContext realmContext && expression.equals("realmIdentifier")) {
-      return realmContext.getRealmIdentifier();
+    if (parameter instanceof RealmId realmId && expression.equals("realmIdentifier")) {
```

Comment:
This class exists only because the old `RealmContext` did not have a meaningful `toString()` representation.

When this class is passed as a method argument to a generated API method:

```java
@Context @MeterTag(key="realm_id",expression="realmIdentifier") RealmContext realmContext
```

I had to use an expression + expression resolver in order to extract the id from `RealmContext`.

Question: is it possible for `RealmId` to have a custom `toString()` representation that simply returns the realm ID? In this case we can get rid of the expression + resolver.

Reply:
Let me check...

Reply:
works

Reply:
Okay, let's at least change the expression from `"realmIdentifier"` to `"id"` – I mean, for now it doesn't matter (it could be any string), but if we ever introduce CEL, then the expression must be valid.

Reply:
shouldn't it be "realm.id" ?

Reply:
Hmm yes, that's probably better 👍